ssh/ssh.go

```diff
@@ -24,25 +26,32 @@ func Launch(cfg *config.Config, env config.Environment, instance aws.EC2Result,
 	lvl := out.GetLevel(env)
 	fmt.Println("")
 	out.Highlight(lvl, "Launching SSH connection to %s", env.Name)
-	out.Highlight(lvl, "[IP: %s | Name: %s | Groups %s | AKA %s", instance.IPAddress, instance.Name, instance.AnsibleGroups, strings.Join(instance.GroupAKA, ", "))
+	out.Highlight(lvl, "[IP: %s | Name: %s | Groups: %s | AKA: %s", instance.IPAddress, instance.Name, instance.AnsibleGroups, strings.Join(instance.GroupAKA, ", "))
 
 	ansibleDir := filepath.Join(cfg.DPSetupPath, "ansible")
+	var userHost string
 	args := []string{"-F", "ssh.cfg"}
-	if portArgs != nil {
-		for _, portArg := range *portArgs {
-			sshPortArgs, err := getSSHPortArguments(portArg)
-			if err != nil {
-				return err
+	if !contains(awsbEnvs, env.Profile) {
+		if portArgs != nil {
+			for _, portArg := range *portArgs {
+				sshPortArgs, err := getSSHPortArguments(portArg)
+				if err != nil {
+					return err
+				}
+				args = append(args, sshPortArgs...)
 			}
-			args = append(args, sshPortArgs...)
 		}
+		userHost = fmt.Sprintf("%s@%s", *cfg.User, instance.IPAddress)
+	} else {
+		os.Setenv("AWS_PROFILE", env.Profile)
+		userHost = fmt.Sprintf("%s@%s", "ubuntu", instance.InstanceId)
```

Comment:
rather than hardcode this user here
I'd be tempted to put the user into the config file, e.g.

```yaml
environments:
  - name: awsb
    user: ubuntu
```

or similar, then

```suggestion
		userHost = fmt.Sprintf("%s@%s", env.User, instance.InstanceId)
```

Reply:
Great idea, didn't think of this. 👍🏻